src/Calculator/GraphingSettings.xaml.cpp

```diff
@@ -0,0 +1,220 @@
+#include "pch.h"
+
+#include "GraphingSettings.xaml.h"
+#include <CalcViewModel\Common\AppResourceProvider.cpp>
+
+using namespace Graphing;
+
+using namespace CalculatorApp;
+using namespace CalculatorApp::Controls;
+
+using namespace Platform;
+using namespace Windows::Foundation;
+using namespace Windows::Foundation::Collections;
+using namespace Windows::UI::Xaml;
+using namespace Windows::UI::Xaml::Controls;
+using namespace Windows::UI::Xaml::Controls::Primitives;
+using namespace Windows::UI::Xaml::Data;
+using namespace Windows::UI::Xaml::Input;
+using namespace Windows::UI::Xaml::Media;
+using namespace Windows::UI::Xaml::Navigation;
+
+GraphingSettings::GraphingSettings()
+{
+    InitializeComponent();
+
+    auto resLoader = AppResourceProvider::GetInstance();
+
+    x_MinHeading->Text = resLoader.GetResourceString(L"x-min");
```

Comment:
If these never change, it would be cleaner and consistent with the rest of the project to assign them in xaml instead by changing the resource name from x-min -> x-min.Text and setting the TextBlock Uid to x:Uid="x-min". Same comment for the rest of this function.

Reply:
Changed.  Interesting, had not used the x:Uid automatic resource replacement before.

Reply:
Done